pages/linux/machinectl.md

```diff
@@ -0,0 +1,25 @@
+# machinectl
+
+> Utility to manage Linux containers, particularly on systems that use the `systemd init` system.
+> It provides a powerful and convenient way to create, start, stop, and manage container instances, referred to as machines in `systemd` terminology.
+> More information: <https://www.freedesktop.org/software/systemd/man/machinectl.html>.
+
+- Create a container:
+
+`sudo machinectl create {{container-name}}`
```

Comment:
This doesn't seem to work:
```
# machinectl create container_name
Unknown command verb create.
```

Reply:
I too can confirm it doesn't work.

```zsh
➜  ~ machinectl create
Unknown command verb create.
➜  ~ sudo machinectl create test
Place your right index finger on the fingerprint reader
Unknown command verb create.
```

Version:

```zsh
➜  ~ machinectl --version
systemd 253 (253.10-1.fc38)
+PAM +AUDIT +SELINUX -APPARMOR +IMA +SMACK +SECCOMP -GCRYPT +GNUTLS +OPENSSL +ACL +BLKID +CURL +ELFUTILS +FIDO2 +IDN2 -IDN -IPTC +KMOD +LIBCRYPTSETUP +LIBFDISK +PCRE2 +PWQUALITY +P11KIT +QRENCODE +TPM2 +BZIP2 +LZ4 +XZ +ZLIB +ZSTD +BPF_FRAMEWORK +XKBCOMMON +UTMP +SYSVINIT default-hierarchy=unified
```